README.md

```diff
@@ -1,2 +1,21 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+
+```Password = The chosen password that the user must find
+UserChosenWord = the word the user clicked/chose
+SameLetters = 0
+numbLetrs = the number of letters in Password
+
+If UserChosenWord =! Password
+	print "Entry denied"
+	for each letter in UserChosenWord and each letter in Password
```

Comment:
This is ambiguous -- not clear that you mean each corresponding letter, for example this could be interpreted as two nested for loops.